src/main/java/io/reactivex/rxjava3/core/Flowable.java

```diff
@@ -14736,6 +14736,8 @@ public final void safeSubscribe(@NonNull Subscriber<@NonNull ? super T> subscrib
      *  <dt><b>Backpressure:</b></dt>
      *  <dd>The operator honors downstream backpressure and expects the current {@code Flowable} to honor backpressure as well.
      *  Violating this expectation, a {@link MissingBackpressureException} <em>may</em> get signaled somewhere downstream.</dd>
+	 *  <dd>The downstream request pattern is not preserved across this operator.
```

Comment:
You don't have to start a new `<dd>` here, just add the text on a new line inside the previous `<dd>`.

Reply:
- [ ] merge `<dd>` sections